bld/namelist_files/namelist_defaults_clm4_5.xml

```diff
@@ -355,7 +355,7 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 <!-- FATES default parameter file                                       -->
 <!-- ================================================================== -->
 
-<fates_paramfile>lnd/clm2/paramdata/fates_params_default_2trop.c190205.nc</fates_paramfile>
+<fates_paramfile>lnd/clm2/paramdata/fates_params_api.7.2.0.nc</fates_paramfile>
```

Comment:
I love having the api version name in the params filename. But, I still think you want the number of PFTs in the name. And I think you want a creation date, or something like it, as you could still have multiple files for the same API right?